ibis/backends/mysql/registry.py

```diff
@@ -110,6 +110,17 @@ def _timestamp_diff(t, expr):
     return sa.func.timestampdiff(sa.text('SECOND'), sa_right, sa_left)
 
 
+def _string_to_timestamp(t, expr):
+    arg, format_str, timezone = expr.op().args
+
+    sa_arg = t.translate(arg)
+    sa_format_str = t.translate(format_str)
+    # if timezone is not None:
+    #    sa_timezone = t.translate(timezone)
```

Comment:
```suggestion
def _string_to_timestamp(t, expr):
    op = expr.op()

    sa_arg = t.translate(op.arg)
    sa_format_str = t.translate(op.format_str)
    # if (timezone := op.timezone) is not None:
    #    sa_timezone = t.translate(timezone)
```
For new translation rules we're moving away from using `.args` on operations because we want rules to not be sensitive to the order in which arguments are listed in the operation class's body.